src/MatBlazor/Components/MatSlider/BaseMatSlider.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
-using System.Threading.Tasks;
+using System.Collections.Generic;
 using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
```

Comment:
We should generally sort the usings alphabetically

Reply:
actually they are sorted, kind of. the only thing is that alphabetical sorting takes place as a second. first are system assembly usings, then package usings and lastly project ones. i think this is the default VS style and in fact most of the present .cs files follows it.